sdk/messaging/azservicebus/changelog.md

```diff
@@ -0,0 +1,4 @@
+# Change Log
```

Comment:
```suggestion
# Release History
```
See https://azure.github.io/azure-sdk/policies_releases.html#change-logs

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-go/pull/15699